Orange/statistics/util.py

```diff
@@ -46,44 +50,72 @@ def bincount(X, max_val=None, weights=None, minlength=None):
             nans = (~nonnan).sum(axis=0)
     else:
         nans = 0. if X.ndim == 1 else np.zeros(X.shape[1], dtype=float)
+
     if minlength is None and max_val is not None:
         minlength = max_val + 1
-    bc = np.array([]) if minlength is not None and minlength <= 0 else \
-        np.bincount(X.astype(np.int32, copy=False),
-                    weights=weights, minlength=minlength).astype(float)
+
+    if minlength is not None and minlength <= 0:
+        bc = np.array([])
+    else:
+        bc = np.bincount(
+            X.astype(np.int32, copy=False), weights=weights, minlength=minlength
+        ).astype(float)
+        # Since `csr_matrix.values` only contain non-zero values, we must count
+        # those separately and set the appropriate bin
+        if sp.issparse(X_):
+            bc[0] = np.prod(X_.shape) - X_.nnz
```

Comment:
Should we maybe make `np.prod(X_.shape) - X_.nnz` as a separate function, something like `sparse_count_zeros`? We already have `_sparse_has_zeros` and we will need to add the number of zero elements to the Table widget sooner or later.

Reply:
I think this is a good idea.

Perhaps it would also make sense to implement `sparse_num_elements` which would essentially be `np.prod(x.shape)` since it's also used a lot. This would make the functionality much clearer, but I'm hesitant because it seems far too excessive.